### PR TITLE
Afterlife bar gets the ':d' deadchat radio prefix

### PR DESCRIPTION
**Pull request by MyNameIsRomayne**
Adds a say prefix for ':d' functionality, restricted to the afterlife bar.

### CODE REVIEW

File: code/modules/speech/prefixes/deadchat.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/datum/say_prefix/deadchat
+	id = ":d"
+
+/// Say prefix for """living""" beings in the afterlife bar/etc. This should not work anywhere else!
+/datum/say_prefix/deadchat/process(datum/say_message/message, datum/speech_module_tree/say_tree)
+	. = message
+
+	var/mob/mob_speaker = message.speaker
+	if (!inafterlife(mob_speaker))
+		return
```
Comment (Mister-Moriarty) [inline code/modules/speech/prefixes/deadchat.dm]: Use `is_compatible_with()` here, in lieu of performing the check in `process()`.

**Review by Mister-Moriarty — CHANGES_REQUESTED** [review]
Do deadchat messages from the afterlife bar send a whisper message too? If not, feel free to `qdel` the original message in the prefix datum.
You may need to add a `QDELETED()` check in `ProcessMessagePrefix()` and the spoken output for the message datum.

File: code/modules/speech/prefixes/deadchat.dm (hunk 1 above)
Comment (MyNameIsRomayne) [reply code/modules/speech/prefixes/deadchat.dm]: done :+1:

**Comment by MyNameIsRomayne** [comment]
image
they do

**Comment by MyNameIsRomayne** [comment]
ah wait crap, nvm

**Comment by MyNameIsRomayne** [comment]
there we are, ready. removed the whisper output

**Pull request merged**